src/util/Utils.cpp

```diff
@@ -503,4 +503,51 @@ void Utils::copyToDirectory(std::string currentPath, std::string copyPath) {
     std::string command = "mkdir -p " +copyPath + "&& " + "cp "+ currentPath + " " + copyPath;
     system(command.c_str());
     
+}
+
+void Utils::editFlagOne(std::string flagPath){
+    
+    std::string filePath = flagPath;
+    ofstream stream;
+    char flag[] = "1";
+
+    stream.open(filePath);
+    if( !stream )
+        cout << "Opening file failed" << endl;
+    stream << flag << endl;
+    if( !stream )
+        cout << "Write failed" << endl;
+
+    stream.close();
+}
+
+void Utils::editFlagZero(std::string flagPath){
+    
+    std::string filePath = flagPath;
+    ofstream stream;
+    char flag[] = "0";
+
+    stream.open(filePath);
+    if( !stream )
```

Comment:
enclose if within {}

Reply:
Fixed in 083074a6c0c0b2f75d2b6a0e81105d64b0a6784d

Reply:
Done